examples/twentyseven-pt-stencil-solver/twentyseven-pt-stencil-solver.cpp

```diff
@@ -0,0 +1,365 @@
+/*******************************<GINKGO LICENSE>******************************
+Copyright (c) 2017-2019, the Ginkgo authors
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+
+1. Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright
+notice, this list of conditions and the following disclaimer in the
+documentation and/or other materials provided with the distribution.
+
+3. Neither the name of the copyright holder nor the names of its
+contributors may be used to endorse or promote products derived from
+this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+******************************<GINKGO LICENSE>*******************************/
+
+#include <array>
+#include <chrono>
+#include <ginkgo/ginkgo.hpp>
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+/*
+double alpha_c = 38.0/6.0;
+double beta_c = -4.0/6.0;
+double gamma_c = -1.0/6.0;
+double delta_c = -1.0/24.0;
+*/
+double alpha_c = 26;
+double beta_c = -1;
+double gamma_c = -1;
+double delta_c = -1;
+std::array<double, 27> coefs;
```

Comment:
Please do the same here as I suggested for the other file:
- Make the variables `constexpr` with the name prefix `default_` or similar to note that these are default values that can be changed in command line
- Remove the global variable `coefs` and call the function with it (`coefs` should be constructed in `main` and then passed by reference to all functions that need it)

Reply:
Same as above. Done